Monika After Story/game/script-topics.rpy

```diff
@@ -11514,8 +11538,43 @@ label monika_grad_speech_call:
     return
 
 label monika_grad_speech_not_paying_attention:
-    #First menu timeout
     hide screen mas_background_timed_jump
+
+    # Make a double check here.
```

Comment:
Please, add a new label for this code instead of changing the old label. That way we can keep the old data about seen labels valid.

Reply:
Understood and [changed](45232dcfa). The label for new code is with a `_doublecheck` now, so the current flow is: 

`monika_grad_speech_call` 
to 
`monika_grad_speech_not_paying_attention_doublecheck` 
to(if no response)
`monika_grad_speech_not_paying_attention`.